src/lib/y2users/user.rb

```diff
@@ -213,15 +219,17 @@ def system=(value)
     def copy
       user = super
       user.password = password.copy if password
+      user.authorized_keys = authorized_keys.map(&:dup)
 
       user
     end
 
     # Validation errors
     #
+    # @param skip [Array<Symbol>] attributes that must be not checked. See {UserValidator#issues}
     # @return [Y2Issues::List]
-    def issues
-      UserValidator.new(self).issues
+    def issues(skip: [])
+      UserValidator.new(self).issues(skip: skip)
```

Comment:
@ancorgs 

Now that I updated this method, I have the doubt if actually `User#issues` should be just `UserValidator#issues(skip: [:password])` directly, without params. I mean

```suggestion
    def issues
      UserValidator.new(self).issues(skip: [:password])
```

I.e, `User#issues` and `User#password_issues` as a shortcuts of `UserValidator#issues(skip: [:password])` and `UserValidator#issues(skip: [:name, :full_name])`

What do you think?

Reply:
As agreed in a video call, let's simply pass down given args. Thanks